src/test/java/org/arquillian/tests/pagetests/BlogPageTest.java

```diff
@@ -81,15 +81,28 @@ public void should_redirect_to_new_announcement_if_banner_is_present() throws Ex
         blogPage.cloudTag()
             .navigate().to("drone");
 
-        blogPage.blogContent()
+        blogPage.releaseBlog()
             .verify()
-            .hasAnnouncementBanner(true);
+                .hasAnnouncementBanner(true);
 
         blogPage.newAnnouncementBanner()
             .navigate().to("Check our latest announcement");
 
-        blogPage.blogContent()
+        blogPage.releaseBlog()
             .verify()
-            .hasAnnouncementBanner(false);
+                .hasAnnouncementBanner(false);
```

Comment:
How about `.hasAnnouncementBanner()` and `.doesNotHaveAnnouncementBanner()` instead?